packages/xml-views-completion/src/providers/attributeValue/literal.ts

```diff
@@ -0,0 +1,51 @@
+import { map, find } from "lodash";
+import { XMLAttribute } from "@xml-tools/ast";
+import { flattenProperties } from "@ui5-language-assistant/logic-utils";
+import { BooleanValueInXMLAttributeValueCompletion } from "../../../api";
+import {
+  getClassByElement,
+  filterMembersForSuggestion
+} from "../utils/filter-members";
+import { UI5AttributeValueCompletionOptions } from "./index";
+
+/**
+ * Suggests boolean value (true/false) inside attribute value
+ */
+export function booleanSuggestions(
+  opts: UI5AttributeValueCompletionOptions
+): BooleanValueInXMLAttributeValueCompletion[] {
+  const xmlElement = opts.element;
+  const xmlAttribute = opts.attribute;
+
+  const elementClass = getClassByElement(xmlElement, opts.context);
+  if (elementClass === undefined) {
+    return [];
+  }
+  const properties = flattenProperties(elementClass);
+  const ui5Property = find(properties, ["name", xmlAttribute.key]);
+  const propType = ui5Property?.type;
+  if (propType?.kind !== "PrimitiveType" || propType.name !== "Boolean") {
+    return [];
+  }
+
+  const values = [
+    {
+      name: "true",
+      value: true
```

Comment:
as mentioned earlier, I am uncertain why we care about the semantic expression value
in a syntactic (completion) context.

would the name suffice?

Reply:
I think there should be a separation between the suggestion name (label) and value. They might not always be the same. The name can be more meaningful than the value when it's not just arbitrary values.